### ISSUE TITLE
USDA_GEMS_TASK1/Image_Crop.ipynb

### ISSUE THREAD

**Issue by runck014** [LRosen656/USDA_GEMS_RGB_COVER#2]
- Separate out function definition code block from the usage of that function in separate cells
- The current headings are hard to follow. Add one for the function / environment setup and another for running the code on the specific imagery dataset.
- Consider only printing every 10 images
- What happens if the file already exists in the folder and the script gets rerun? Should it overwrite the file or skip it?

**Comment by LRosen656** [comment]
All resolved except final point.

**Comment by LRosen656** [comment]
Will overwrite the files, for spreadsheets, I have old files moved to archived.
